initramfs-tools/README.md

```diff
@@ -1,13 +1,33 @@
 ## Initramfs-tools based systems(Ubuntu and derivatives)
 
-After installation generate your credentials and add keys to your disk as described in the top-level README
-then add `initramfs,keyscript=fido2luks` to your `/etc/crypttab`
+[Dowload and install the precompiled deb from releases.](https://github.com/shimunn/fido2luks/releases)
```

Comment:
Linking the deb is a good idea however it be good to hint that installation from source is possible